packages/fluentui/react-northstar/src/components/Input/Input.tsx

```diff
@@ -431,7 +431,7 @@ Input.defaultProps = {
   wrapper: {},
   iconPosition: 'end',
   errorIndicator: <ExclamationCircleIcon />,
-  successIndicator: <PresenceAvailableIcon />,
+  successIndicator: <CheckmarkCircleIcon />,
```

Comment:
This changes the Icon and doesn't match the design spec...

Your PR:

<img width="215" alt="Screen Shot 2021-01-19 at 9 54 32" src="https://user-images.githubusercontent.com/8545105/105037293-87ee2180-5a3c-11eb-96d8-2b98a1af9c09.png">


Design Spec:

![Uploading Screen Shot 2021-01-19 at 9.54.45.png…]()

Reply:
![image](https://user-images.githubusercontent.com/14893575/105050168-2cd92080-5a76-11eb-8d01-45bae3a8a2ed.png)

![image](https://user-images.githubusercontent.com/14893575/105050209-36fb1f00-5a76-11eb-9f14-ee5f341127de.png)

Presence icon is meant for presence

Reply:
Forgot to add "outline" prop there in PR

Reply:
Fixed that one liner. Should look ok now @assuncaocharles

Reply:
![image](https://user-images.githubusercontent.com/14893575/105479365-0a7d1800-5cad-11eb-9745-7a6f18d3d49d.png)